### PR TITLE
store/tikv: move rawkv_test to /tests

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?
store/tikv has dependencies on `mockstore` and `unistore` because tests need to use them. We can resolve it by moving all tests to standaline package.

Part of https://github.com/pingcap/tidb/issues/22513

### What is changed and how it works?
What's Changed:
- move test files into `store/tikv/tests`

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

File: store/tikv/tests/rawkv_test.go [hunk 1]
```diff
@@ -188,6 +187,8 @@ func (s *testRawKVSuite) TestSimple(c *C) {
 	c.Assert(err, NotNil)
 }
 
+var rawBatchPutSize = tikv.ConfigProbe{}.GetRawBatchPutSize()
```
Comment (andylokandy) [inline store/tikv/tests/rawkv_test.go:190]: I suggest replace https://github.com/pingcap/tidb/blob/b5fab43a93d3594d08acd3eb5d078bee2a7e40f1/store/tikv/tests/rawkv_test.go#L197 with `tikv.ConfigProbe{}.GetRawBatchPutSize()` and remove this definition.

Comment (disksing) [reply store/tikv/tests/rawkv_test.go:190]: ok

**Review by andylokandy — APPROVED** [review]

**Pull request merged**